src/main/java/io/codelex/Test/Exercise3/Joiner.java

```diff
@@ -0,0 +1,30 @@
+package io.codelex.Test.Exercise3;
+
+import java.util.List;
+
+public class Joiner<T> {
+    String sep;
+
+    String result;
```

Comment:
Metodei join uzreiz ir jāatgriež String rezultāts, šis mainīgais attiecīgi ir lieks.